java/source/org/libreoffice/ide/eclipse/java/build/messages.properties

```diff
@@ -3,6 +3,6 @@ JODContainerPage.Title=LibreOffice UNO Connector
 OOoClasspathContainer.LibrariesName={0} libraries
 
 OOoContainerPage.DialogDescription=LibreOffice Library edition page
-OOoContainerPage.DialogImage=/icons/library_wiz.png
+OOoContainerPage.DialogImage=/icons/library_wiz.gif
 OOoContainerPage.DialogTitle=LibreOffice Libraries
 OOoContainerPage.ClasspathSetFailed=Error while setting the project classpath
```

Comment:
In the messages.properties file [OOoContainerPage.DialogImage=/icons/library_wiz.png]. When you gor for Right-Click on Java Uno Project -> Java Build Path -> Libraries ->  LibreOffice _._ libraries -> Edit
A wiard comes up, this is the result of OOoContainerPage. [Originally top right corner image is missing] The message.properties has the location of the image, but in that location no image of that name existed. Therefore I downloaded the Eclipse icon pack from 
https://git.eclipse.org/c/platform/eclipse.platform.images.git/

TAG             DOWNLOAD
R4_6            eclipse.platform.images-R4_6.zip

inside this I used the icon:
/eclipse.platform.images-R4_6/org.eclipse.images/eclipse-gif/org.eclipse.jdt.ui/icons/full/obj16/library_obj.gif  and renamed it to  'library_wiz.gif' and placed it under the icons folder inside Java module